prow/config/jobs/api-1.6.yaml

```diff
@@ -0,0 +1,52 @@
+branches:
+- release-1.6
+image: gcr.io/istio-testing/build-tools:master-latest
```

Comment:
this will change immediately after right? since we should use a pinned version normally